pilot/pkg/model/service.go

```diff
@@ -204,8 +204,8 @@ func ParseProtocol(s string) Protocol {
 		return ProtocolHTTP2
 	case "https":
 		return ProtocolHTTPS
-	case "tcp_tls":
-		return ProtocolTCPTLS
+	case "tls":
```

Comment:
See above - we can't break configs, if you must - make sure both old name and new name are supported.

Reply:
and tcp_tls went in after 0.8